app/plots.py

```diff
@@ -26,44 +26,95 @@ def get_ratings(shortname, rating_type='elo'):
 
 def plot_ratings(shortname, rating_type):
     source = get_ratings(shortname, rating_type=rating_type)
-    hover_text = []
-    for index, row in source.iterrows():
-        match_id = row['match_id']
-        if pd.isnull(match_id):
-            # To allow formatting of ids laters
-            match_id = 0
-        hover_text.append((
-            "Match ID: {match_id}<br>"+
-            "Time: {date}<br>"+
-            "Rating: {rating}<br>").format(
-                match_id = format(match_id, "1.0f") ,
-                date = row["date"].ctime(),
-                rating = format(row["rating"], "1.0f")
-            ))
-    
-    data = go.Scatter(
-            x = source.loc[:,"date"],
-            y = source.loc[:,"rating"],
-            text = hover_text,
-            hoverinfo = 'text',
-            marker = dict(
-                color = 'green'
-            ),
-            showlegend = False
-        )
-    data_comp = [data]
 
-    layout_comp = go.Layout(
-        title='Performance over time',
-        hovermode='closest',
-        xaxis=dict(
-            title='Time',
-        ),
-        yaxis=dict(
-            title='Rating',
-    ))
+    if len(source)<7:
```

Comment:
Why 7? What is the magic number?

Reply:
It's a bit arbitrary. I changed it so that it's now a constant that is defined above the if statement